runtime/compiler/z/codegen/J9TreeEvaluator.cpp

```diff
@@ -2238,14 +2238,31 @@ J9::Z::TreeEvaluator::inlineMathFma(TR::Node *node, TR::CodeGenerator *cg)
    TR_ASSERT_FATAL(node->getNumChildren() == 3,
    "In function inlineMathFma, the node at address %p should have exactly 3 children, but got %u instead", node, node->getNumChildren());
 
-   TR::Register      * targetRegister      = cg->allocateRegister(TR_FPR);
-
    TR::Register      * v1      = cg->evaluate(node->getFirstChild());
    TR::Register      * v2      = cg->evaluate(node->getSecondChild());
-   TR::Register      * v3      = cg->evaluate(node->getThirdChild());
+   TR::Register      * v3;
```

Comment:
As we are touching this evaluator, I would recommend to use variable names which makes it easier to correlate to the API it is inlining [1]

[1]. https://docs.oracle.com/javase/9/docs/api/java/lang/Math.html#fma-double-double-double-